packages/headless/src/features/generated-answer/generated-answer-slice.ts

```diff
@@ -49,8 +53,11 @@ export const generatedAnswerReducer = createReducer(
         state.liked = false;
         state.disliked = true;
       })
-      .addCase(resetAnswer, () => {
-        return getGeneratedAnswerInitialState();
+      .addCase(resetAnswer, (state) => {
+        return {
+          ...getGeneratedAnswerInitialState(),
```

Comment:
I deconstructed this in my branch to make it explicit because I got very confused for a good 30 minutes trying to figure out why my state change was being overridden before executing the query.
I think at this point it would be safer to be explicit about what we want to reset rather than have to start omitting what we don't want to reset. What do you think?

Reply:
I've been thinking about it for 12 minutes and i'm not even sure anymore

Reply:
nvm i changed my mind